bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/SpanImpl.kt

```diff
@@ -27,6 +27,16 @@ class SpanImpl internal constructor(
 
     override val attributes: Attributes = Attributes()
 
+    /**
+     * Internally SpanImpl objects can be chained together as a fast linked-list structure
+     * (nicknamed [SpanChain]) allowing us a lock-free / allocation-free batching structure.
+     *
+     * See [Tracer] for more information on how this is used.
+     */
+    @JvmField
+    @JvmSynthetic
```

Comment:
Does this need to be synthetic since it's in the internal package?

Reply:
Possibly not, but in this case we want to be really certain that it can't be tampered with.